Bootstrap/app/src/main/res/values/dimens.xml

```diff
@@ -91,11 +91,11 @@
 
     <dimen name="wLogo_layout_height">56dp</dimen>
     <dimen name="wlogo_layout_width">56dp</dimen>
-    <dimen name="wlogo_marginStart">16dp</dimen>
+    <dimen name="wlogo_marginStart">0dp</dimen>
 
 
-    <dimen name="home_woloxText_marginStart">72dp</dimen>
-    <dimen name="home_woloxText_marginTop">16dp</dimen>
+    <dimen name="home_woloxText_marginStart">0dp</dimen>
```

Comment:
A margin of 0dp has no sense in this case. Sometimes you will need yo use 0dp as width or height but in that cases you can write the number on the xml of the fragment / view .  it's the only exception